cypress/integration/logistics/materialentnahme_partial_HU.js

```diff
@@ -0,0 +1,244 @@
+      productForPackingMaterialLU,
+      productType,
+      '24_Gebinde'
+    );
+  });
+  it('create packing material for handling unit - Transport unit', function() {
+    cy.fixture('product/packing_material.json').then(packingMaterialJson => {
+      Object.assign(new PackingMaterial(), packingMaterialJson)
+        .setName(packingMaterialForTU)
+        .setProduct(productForPackingMaterialTU)
+        .setLength('0')
+        .setWidth('0')
+        .setHeight('0')
+        .apply();
+    });
+  });
+  it('create packing material for handling unit - Load unit', function() {
+    cy.fixture('product/packing_material.json').then(packingMaterialJson => {
+      Object.assign(new PackingMaterial(), packingMaterialJson)
+        .setName(packingMaterialForLU)
+        .setProduct(productForPackingMaterialLU)
+        .setLength('0')
+        .setWidth('0')
+        .setHeight('0')
+        .apply();
+    });
+  });
+  it('create packing instruction for handling unit - Transport unit', function() {
+    cy.fixture('product/packing_instructions.json').then(packingInstructionsJson => {
+      Object.assign(new PackingInstructions(), packingInstructionsJson)
+        .setName(packingInstructionsNameForTU)
+        .apply();
+    });
+  });
+  it('create packing instruction for handling unit - Load unit', function() {
+    cy.fixture('product/packing_instructions.json').then(packingInstructionsJson => {
+      Object.assign(new PackingInstructions(), packingInstructionsJson)
+        .setName(packingInstructionsNameForLU)
+        .apply();
+    });
+  });
+  it('create packing instruction version for handling unit - Transport unit; with packgut - quantity=0.00 and packmittel', function() {
+    cy.fixture('product/packing_instructions_version.json').then(pivJson => {
+      Object.assign(new PackingInstructionsVersion(), pivJson)
+        .setName(packingInstructionsVersionForTU)
+        .setPackingInstructions(packingInstructionsNameForTU)
+        .setPackingMaterial(packingMaterialForTU)
+        .apply();
+    });
+    /**For some reasons,the quantity for the packgut cannot be added in advanced edit mode, only like below */
+    cy.selectTab('M_HU_PI_Item');
+    cy.selectNthRow(1)
+      .find('.Quantity')
+      .dblclick({ force: true })
+      .find('.form-field-Qty input')
+      .type('0', { force: true });
+    cy.selectTab('M_HU_PI_Item');
+  });
+  it('create packing instruction version for handling unit - Loading unit; with packmittel and packingInstructionsNameForTU as unter-packvorschrift', function() {
+    cy.fixture('product/packing_instructions_version.json').then(pivJson => {
+      Object.assign(new PackingInstructionsVersion(), pivJson)
+        .setName(packingInstructionsVersionForLU)
+        .setPackingInstructions(packingInstructionsNameForLU)
+        .setPackingMaterial(packingMaterialForLU)
+        .setUnit('Load/Logistique Unit')
+        .apply();
+    });
+    cy.selectTab('M_HU_PI_Item');
```

Comment:
Same request as above:

Imo, this should be added to `PackingInstructionsVersion.apply_M_HU_PI_Item()`, since maybe other tests will need to set it as well. (please find a better method name :D)